tests/integration/steps_downtime.go

```diff
@@ -126,14 +133,16 @@ func stepsDowntime(consumerName string) []Step {
 					ValPowers: &map[validatorID]uint{
 						// Non faulty validators still maintain just above 2/3 power here
 						validatorID("alice"): 509,
-						validatorID("bob"):   495,
+						validatorID("bob"):   500,
+						// Carol's stake should be slashed
```

Comment:
Is this suggestion correct?
```suggestion
						// Carol's stake should be jailed
```